src/main/java/se/sundsvall/supportmanagement/api/model/errand/Errand.java

```diff
@@ -23,18 +25,15 @@
 import se.sundsvall.supportmanagement.api.validation.groups.OnCreate;
 import se.sundsvall.supportmanagement.api.validation.groups.OnUpdate;
 
-import io.swagger.v3.oas.annotations.media.ArraySchema;
-import io.swagger.v3.oas.annotations.media.Schema;
-
 @Schema(description = "Errand model")
 public class Errand {
 
 	@Schema(description = "Unique id for the errand", example = "f0882f1d-06bc-47fd-b017-1d8307f5ce95", accessMode = READ_ONLY)
-	@Null(groups = {OnCreate.class, OnUpdate.class})
+	@Null(groups = { OnCreate.class, OnUpdate.class })
 	private String id;
 
 	@Schema(description = "Unique number for the errand", example = "KC-23010001", accessMode = READ_ONLY)
-	@Null(groups = {OnCreate.class, OnUpdate.class})
+	@Null(groups = { OnCreate.class, OnUpdate.class })
 	private String errandNumber;
```

Comment:
Might not be in scope of this story, but validating something this is READ_ONLY seems a bit excessive.

Reply:
It is "read_only" in the API, but this is not enforced (it's only an documentation-thingy). 
You can still send the errandNumber in the request, if you want to. 

I guess that's the idea behind this validation?